components/automate-ui/src/app/helpers/auth/sorter.ts

```diff
@@ -0,0 +1,29 @@
+
+export class ChefSorters {
+
+    // normalSort takes in an Object and returns an Array of Objects
+    // sorted by the property chosen.
+
+    public static normalSort(input: object, property: string): Array<any> {
+        const sortedArray = Object.values(input);
+
+        // options for localeCompare
+        const opts = {
```

Comment:
Unsure if it would be cleaner to just write the options object in the localeCompare on line 21 since it's only used once and is pretty small.

Reply:
That's fine; your choice.